app/assets/stylesheets/application.scss

```diff
@@ -42,3 +42,11 @@
   }
   thead {background: $light-grey}
 }
+
+.td-numeric,.th-numeric {
+  text-align: right;
+}
+
+.td-date,.th-date {
+  text-align: center;
+}
```

Comment:
Minor issue, if you wouldn't mind indulging me:

Instead of `.td-date`, `th-date`, `td-numeric`, `th-numeric` could you do: `td.date`, `th.date`, `td.numeric` `th.numeric`? Alternately, I *think* you could do `table .numeric` (note the space) and `table .date`

The class to be applied, below, would then be:
`<td class="date">`, `<th class="numeric"` etc.